config/exceptions.py

```
@@ -0,0 +1,21 @@
+
+    if isinstance(exc, BlogException):
+        return handle_blog_exceptions(exc, context)
+
+    # DRF 기본 핸들러
+    response = exception_handler(exc, context)
+    if response is not None:
+        return response
+    return None
+
+
+def handle_blog_exceptions(exc, context):
```

Ruff (0.15.5)

[warning] Unused function argument: `context`

(ARG001)